docs/styles/_newsletter.scss

```diff
@@ -0,0 +1,3 @@
+.newsletter {
+    background: #333333;
```

Comment:
please use `$gray-dark` variable

Reply:
@szafran89 don't see this border here https://i.gyazo.com/ebca75029d9d9925f2c807603410ebc5.png (desktop version)

but I will prepare another variation for mobile

Reply:
done